frontend/src/pages/modelServing/screens/metrics/BiasMetricConfigSelector.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {
+  Select,
+  SelectGroup,
+  SelectOption,
+  SelectOptionObject,
+  SelectVariant,
+} from '@patternfly/react-core';
+import { useExplainabilityModelData } from '~/concepts/explainability/useExplainabilityModelData';
+import { BiasMetricConfig } from '~/concepts/explainability/types';
+import { MetricTypes } from '~/api';
+import { byId, byNotId } from '~/pages/modelServing/screens/metrics/utils';
+
+type BiasMetricConfigSelector = {
+  onChange?: (x: BiasMetricConfig[]) => void;
+  initialSelections?: BiasMetricConfig[];
+};
+
+const BiasMetricConfigSelector: React.FC<BiasMetricConfigSelector> = ({
+  onChange,
+  initialSelections,
+}) => {
+  const { biasMetricConfigs, loaded } = useExplainabilityModelData();
+
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState<BiasSelectOption[]>(() => {
+    if (initialSelections) {
+      return initialSelections.map(createBiasSelectOption);
+    }
+    return [];
+  });
+
+  const elementId = React.useId();
+
+  const changeState = React.useCallback(
+    (options: BiasSelectOption[]) => {
+      setSelected(options);
+      onChange && onChange(options.map((x) => x.biasMetricConfig));
+    },
+    [onChange],
+  );
+
+  return (
+    <>
+      <span id={elementId} hidden>
+        Select the metrics to display charts for
+      </span>
+      <Select
```

Comment:
Is it possible to limit the width of this Select component? The layout of the toolbar will be broken if we choose a lot of selections and show them all:

<img width="1492" alt="Screenshot 2023-06-09 at 4 43 03 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/f3d5d6c1-947d-4b6c-bfcd-615ebef8deb4">

Reply:
Really good point, I've limited it to 600px now, that seems like a good compromise, let me know what you think.